app/code/Magento/Directory/Model/Observer.php

```diff
@@ -83,12 +88,13 @@ public function __construct(
     }
 
     /**
-     * @param mixed $schedule
+     * Schedule update currency rates
+     *
      * @return void
      * @throws \Exception
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function scheduledUpdateCurrencyRates($schedule)
+    public function scheduledUpdateCurrencyRates()
```

Comment:
Unfortunately, we are not able to change the public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)